src/lib.rs

```diff
@@ -117,6 +117,31 @@ impl HidApi {
         })
     }
 
+    /// Initializes the hidapi.
+    /// it skips device scanning.
+    pub fn new_without_enumerate() ->HidResult<Self> {
+        if cfg!(any(
```

Comment:
This does not actually work, the calls to libusb are still compiled, meaning this breaks all platforms except libusb based ones. See CI run.

Solve it like in this example, where the code is not actually compiled: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=18dd84fabb7e4531d0d7e0d93031be9f